kde-apps/krdc/patches/krdc-24.02.2.patchset

```diff
@@ -0,0 +1,117 @@
+From 52535261201ddc351862e14f0c9116c1fe1c3dd6 Mon Sep 17 00:00:00 2001
+From: Sergei Reznikov <diver@gelios.net>
+Date: Fri, 24 Aug 2018 17:55:51 +0300
+Subject: Fix build on Haiku
+
+
+diff --git a/cmake/modules/FindLibSSH.cmake b/cmake/modules/FindLibSSH.cmake
+index c1fecfd..9dc6669 100644
+--- a/cmake/modules/FindLibSSH.cmake
++++ b/cmake/modules/FindLibSSH.cmake
+@@ -17,6 +17,8 @@ find_path(LIBSSH_INCLUDE_DIR
+     /usr/local/include
+     /opt/local/include
+     /sw/include
++	/system/develop/headers
++	/system/develop/headers/x86
+     ${CMAKE_INCLUDE_PATH}
+     ${CMAKE_INSTALL_PREFIX}/include
+ )
+@@ -30,6 +31,8 @@ find_library(SSH_LIBRARY
+     /usr/local/lib
+     /opt/local/lib
+     /sw/lib
++	/system/develop/headers
++	/system/develop/headers/x86
+     ${CMAKE_LIBRARY_PATH}
+     ${CMAKE_INSTALL_PREFIX}/lib
+ )
+diff --git a/vnc/CMakeLists.txt b/vnc/CMakeLists.txt
+index 20f74f0..6dd2933 100644
+--- a/vnc/CMakeLists.txt
++++ b/vnc/CMakeLists.txt
+@@ -31,6 +31,10 @@ target_link_libraries(krdc_vncplugin
+     krdccore
+ )
+ 
++if(HAIKU)
++    target_link_libraries(krdc_vncplugin network)
++endif ()
++
+ if(QT_MAJOR_VERSION STREQUAL "6")
+     target_link_libraries(krdc_vncplugin KF6::KCMUtils)
+ endif()
+diff --git a/vnc/vncclientthread.cpp b/vnc/vncclientthread.cpp
+index 5b5804c..cf96aed 100644
+--- a/vnc/vncclientthread.cpp
++++ b/vnc/vncclientthread.cpp
+@@ -668,7 +668,7 @@ void VncClientThread::clientSetKeepalive()
+     }
+ 
+     optval = m_keepalive.intervalSeconds;
```

Comment:
should be #ifdef TCP_KEEPIDLE

Reply:
Will check, left-over from previous patch.

Reply:
Fixed, thanks!